### ISSUE TITLE
Broken master build (re. mariaDB4j-maven-plugin integration tests)

### ISSUE THREAD

**Issue by vorburger** [vorburger/MariaDB4j#262]
Recent PRs such as #258 and #260 illustrate that the master build is broken:

```
[INFO] --- maven-invoker-plugin:3.2.0:run (integration-test) @ mariaDB4j-maven-plugin ---
[INFO] Building: mariadb4j-maven-plugin-test-basic/pom.xml
[INFO] [INFO] Scanning for projects...
[INFO] [INFO] Downloading from local.central: file:///home/travis/.m2/repository/ch/vorburger/mariaDB4j/mariaDB4j-pom/2.4.0-SNAPSHOT/maven-metadata.xml
[INFO] [INFO] Downloading from local.central: file:///home/travis/.m2/repository/ch/vorburger/mariaDB4j/mariaDB4j-pom/2.4.0-SNAPSHOT/mariaDB4j-pom-2.4.0-SNAPSHOT.pom
[INFO] [ERROR] [ERROR] Some problems were encountered while processing the POMs:
[INFO] [FATAL] Non-resolvable parent POM for ch.vorburger.mariaDB4j:mariadb4j-maven-plugin-test-basic:[unknown-version]: Could not find artifact ch.vorburger.mariaDB4j:mariaDB4j-pom:pom:2.4.0-SNAPSHOT in local.central (file:///home/travis/.m2/repository) and 'parent.relativePath' points at wrong local POM @ line 6, column 13
[INFO]  @ 
[INFO] [ERROR] The build could not read 1 project -> [Help 1]
[INFO] [ERROR]   
[INFO] [ERROR]   The project ch.vorburger.mariaDB4j:mariadb4j-maven-plugin-test-basic:[unknown-version] (/home/travis/build/vorburger/MariaDB4j/mariaDB4j-maven-plugin/target/it/mariadb4j-maven-plugin-test-basic/pom.xml) has 1 error
[INFO] [ERROR]     Non-resolvable parent POM for ch.vorburger.mariaDB4j:mariadb4j-maven-plugin-test-basic:[unknown-version]: Could not find artifact ch.vorburger.mariaDB4j:mariaDB4j-pom:pom:2.4.0-SNAPSHOT in local.central (file:///home/travis/.m2/repository) and 'parent.relativePath' points at wrong local POM @ line 6, column 13 -> [Help 2]
[INFO] [ERROR] 
[INFO] [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[INFO] [ERROR] Re-run Maven using the -X switch to enable full debug logging.
[INFO] [ERROR] 
[INFO] [ERROR] For more information about the errors and possible solutions, please read the following articles:
[INFO] [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[INFO] [ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException
[INFO]   The build exited with code 1. See /home/travis/build/vorburger/MariaDB4j/mariaDB4j-maven-plugin/target/it/mariadb4j-maven-plugin-test-basic/build.log for details.
[INFO]           mariadb4j-maven-plugin-test-basic/pom.xml ........ FAILED (2.0 s)
[INFO] Building: start-stop/pom.xml
```

Would anyone watching this be willing to step up and help maintaing this project, see #213 ?

@srbala  @glittle1972  @mdindoffer  @yiftizur @jtmuhone

**Comment by mdindoffer** [comment]
I believe this is already fixed by my PR from April https://github.com/vorburger/MariaDB4j/pull/248

As for the patronage I'd have to think about it, but there are probably better candidates.

**Comment by vorburger** [comment]
> I believe this is already fixed by my PR from April #248

@mdindoffer indeed it does - but when I originally glanced over your PR I hadn't understood why..

Just FTR, the problem here was that we're at `<version>2.4.1-SNAPSHOT` everywhere, but somehow `mariaDB4j-maven-plugin/src/it/mariadb4j-maven-plugin-test-basic/pom.xml` was left behind.

It would be nice if this was caught earlier in the future; I wonder if #263 will help to prevent this.
